packages/machine/package.json

```diff
@@ -14,6 +14,7 @@
     "build": "tsc -p tsconfig.test.json && rollup -c",
     "test": "tsc -b && jest --runInBand --detectOpenHandles --bail",
     "test-debug": "node --inspect-brk jest --runInBand",
+    "test-debug-ide": "node $NODE_DEBUG_OPTION ./node_modules/.bin/jest --runInBand",
```

Comment:
This is really useful for those of us using JetBrains IDEs as it allows you to use the visual debugger. Huge productivity boost

Reply:
Does this work with VS Code too?

Reply:
Not sure.